src/stories/2-frontend-engine/validation/validation-story-component.tsx

```diff
@@ -187,20 +187,20 @@ const Wrapper = styled.div`
 	max-width: 90vw;
 
 	code {
-		border-radius: 4px;
+		border-radius: ${Radius.sm};
 	}
 `;
 
 const Section = styled.section`
 	display: flex;
 	flex-direction: column;
-	gap: 0.5rem;
-	margin-bottom: 2rem;
-	padding: 1.25rem;
+	gap: ${Spacing["spacing-8"]};
+	margin-bottom: ${Spacing["spacing-32"]};
+	padding: ${Spacing["spacing-20"]};
 
 	// follow storybook styling
-	border: 1px solid rgba(38, 85, 115, 0.15);
-	border-radius: 4px;
+	border: ${Border["width-010"]} ${Border.solid} rgba(38, 85, 115, 0.15);
+	border-radius: ${Radius.sm};
```

Comment:
this can be reverted as it's meant to follow an external style and shouldn't be affected by themes